### ISSUE TITLE
[Community Help] Renaming 'heat'

### ISSUE THREAD

**Issue by 1313e** [1313e/CMasher#23]
TL;DR: [Vote here](https://www.rkursem.com/poll/view.php?id=7a534757f91cab5f1)

So, one of the very first colormaps I have made, before CMasher existed or was even an idea, is the `heat` colormap.
Even though the `rainforest` colormap is the very first colormap I have made and will therefore always be special to me, the `heat` colormap is probably a close second.
I personally really like this colormap, and judging from feedback I get from others and papers that use CMasher, so do a lot of others.

Now, the problem I have with `heat` is its name.
I try to give every colormap a name that provides a good first impression to the user, such that you don't need to know the colormap in order to have a rough idea of what it would look like.
The `heat` colormap is meant to look like the typical gas flame you get from a burner or stove.
I initially wanted to call it `thermal`, but because MPL already has a colormap with this name (and my colormaps did not use name prefixes yet), I could not do that.
So, I settled for `heat`.

I however think that `heat` does not really set a good expectation for the colormap.
Therefore, I have been wanting to change its name for a long time now, but given its popularity, changing its name would break some codes.
Plus, I am not entirely sure what to change it to actually.

So, what I want to do, is ask everyone reading this issue to either vote on a name, or come with new names.
You can do so [here](https://www.rkursem.com/poll/view.php?id=7a534757f91cab5f1), where you can also add new options for users to vote on.
If enough people vote (not entirely sure yet what I define as 'enough'), I will rename `heat` to whatever was picked the most.

**Comment by 1313e** [comment]
Currently, it seems that `torch` is preferred as the replacement name for `heat`.

**Comment by 1313e** [comment]
After having this poll open for way too long, I have made the decision to rename `heat` to `torch`.
This will be done in v1.6.1.
`heat` itself will remain available as well (with a warning that it was removed) until v1.7 is released.
